nixos/modules/services/security/e-imzo.nix

```
@@ -0,0 +1,39 @@
+{ config, lib, pkgs, ... }:
+let cfg = config.services.e-imzo;
+in {
+  options = with lib; {
+    services.e-imzo = {
+      enable = mkEnableOption "E-IMZO";
+
+      package = mkPackageOption pkgs "e-imzo" {
```

```suggestion
  options = {
    services.e-imzo = {
      enable = lib.mkEnableOption "E-IMZO";

      package = lib.mkPackageOption pkgs "e-imzo" {
```
We try to avoid `with lib;` in new code.

---

I'm terribly sorry, I assumed it from other modules.